.github/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+# See: https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Interesting. Why add the complyscribe entries here?

Reply:
Since dependabot ignores absent directories, it was to simplify the logic of the synchronization script. Otherwise we would need multiple files in org-infra or a mechanism to treat the particularities or each repository. There is a place for an improvement on this in the sync script. Or we can even remove the dependabot file from the sync once all the repositories are configured.